### DISCUSSION TITLE
Feature request: support validation attribute in json schema

### DISCUSSION THREAD

**Discussion by slevy85** [johngeorgewright/runtyping#430] [Ideas]
Hi and thank you fr this great library.

The json schema provides some validation attributes like minLength, maxLength and others, it would be nice to have them reflected in the generated code also.

What do you think ?

**Comment by johngeorgewright** [comment]
I agree this would be a good idea. However, we're currently very, very far from being able to implement this. This is because we have only written the `TS -> [validator]` part of the problem. And instead of creating something that would read JSONSchema and output a `validator`, we have simply used another library (https://github.com/bcherny/json-schema-to-typescript) to convert JSONSchema to TypeScript first. I don't really intend on reinventing the wheel, unless absolutely necessary.

I have another ticket open (#347) to take JSdocs in to consideration. Maybe json-schema-to-typescript could add JSdocs..? Might not be something they'd support, but it would be far simpler for us.
